src/mopeka.c

```diff
@@ -12,14 +12,14 @@
 #include "mopeka.h"
 #include "task.h"
 
-#define HW_ID_LPG	 3     // Pro Check LPG bottom-up
-#define HW_ID_P200	 4     // Pro-200 top-down
-#define HW_ID_H2O	 5     // Pro Check H2O bottom-up
-#define HW_ID_PPB    8     // PRO+ top-down Boosted BLE sensor
-#define HW_ID_PPC    9     // PRO+ top-down Cellular Sensor
-#define HW_ID_TDB  0x0A  // TD-40 Top-down Boosted BLE sensor
-#define HW_ID_TDC  0x0B  // TD-40 Top-down Cellular Sensor
-#define HW_ID_UNIV   0x0C  // Pro Check Universal Sensor
+#define HW_ID_PRO                3     // Pro Check LPG bottom-up
```

Comment:
I'll go ahead and use the updated IDs but I think I was more descriptive last time and Victron simplified the names of the constants.  They'll also probably pull out all of the comments so, while I appreciate you cleaning these up, I don't know how much of that will make it into the final code.  It's up to Victron.